pyisy/nodes/node.py

```diff
@@ -136,6 +138,15 @@ def formatted(self):
         """Return the formatted value with units, if provided."""
         return self._formatted
 
+    @property
+    def is_battery_node(self):
+        """
+        Confirm if this is a battery node or a normal node.
+
+        Battery nodes do not provide a 'ST' property, only 'BATLVL'.
+        """
+        return self._is_battery_node
```

Comment:
Are there other node types that lack `ST`?

Reply:
They are the only ones that we explicitly set a ST with if missing:

```python
        if prop_id == PROP_STATUS:
            state = result
            state_set = True
        elif prop_id == PROP_BATTERY_LEVEL and not state_set:
            state = result
```